src/views/shared/AuthContainer.js

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define([
+  'okta'
+],
+function (Okta) {
+  var CAN_REMOVE_BEACON_CLS = 'can-remove-beacon';
+  return Okta.View.extend({
+    className: 'auth-container main-container',
+    id: 'okta-sign-in',
+    attributes: { 'data-se': 'auth-container' },
```

Comment:
i don't think we need a data-se here right now since this isn't being used in a selenium test